WcaOnRails/config/locales/en.yml

```diff
@@ -42,7 +42,6 @@ en:
     ST: "São Tomé and Príncipe"
     SZ: "Eswatini"
     TT: "Trinidad and Tobago"
-    VA: "Holy See"
```

Comment:
Wouldn't it require to rename Holy See but not removing it?

Reply:
No, because the gem we use to provide country name translation (i18n-country-translations) uses "Vatican City" as well.
If it ever changes, we have a test checking that our 3 different sources of country names (db, wca-states.json, that i18n gem) are consistent, so we would know about it!